ogr/abstract.py

```diff
@@ -1542,6 +1558,15 @@ def add_group(self, group: str, access_level: AccessLevel) -> None:
         """
         raise NotImplementedError
 
+    def remove_group(self, group: str) -> None:
+        """
+        Remove group from project.
+
+        Args:
+            group: Name of the group.
+        """
+        raise NotImplementedError
```

Comment:
```suggestion
        raise NotImplementedError()
```

Reply:
Ok, let me fix also the one above :slightly_smiling_face: